### ISSUE TITLE
oxAuth authorization page throws error when French locale is used for it

### ISSUE THREAD

**Issue by aliaksander-samuseu** [GluuFederation/oxAuth#827]
Environment:
CentOS 7.4, Gluu CE 3.1.3

Issue in a nutshell (exact steps to reproduce it are unkonw, but one user somehow managed to trigger it; but its cause is now clear):

French localization file located at `/WEB-INF/classes/oxauth_fr.properties` in oxAuth.war contains dubious command in one of strings used on the oxAuth's authz page: `authorize.requestingPermissionForScopes=#{client.getClientName()}demande l'autorisation de faire ce qui suit:`

For reference, here is how it looks in comparison with the same sting in other locales:
```
grep -i -r -e requestingPermissionForScopes /opt/jetty-9.4/temp/jetty-localhost-8081-oxauth.war-_oxauth-any-7004841259582193821.dir/webapp/
/opt/jetty-9.4/temp/jetty-localhost-8081-oxauth.war-_oxauth-any-7004841259582193821.dir/webapp/WEB-INF/incl/layout/authorize-extended-template.xhtml:							<h:outputFormat value="#{msgs['authorize.requestingPermissionForScopes']}">
/opt/jetty-9.4/temp/jetty-localhost-8081-oxauth.war-_oxauth-any-7004841259582193821.dir/webapp/WEB-INF/classes/oxauth_fr.properties:authorize.requestingPermissionForScopes=#{client.getClientName()}demande l'autorisation de faire ce qui suit:
/opt/jetty-9.4/temp/jetty-localhost-8081-oxauth.war-_oxauth-any-7004841259582193821.dir/webapp/WEB-INF/classes/oxauth_ru.properties:authorize.requestingPermissionForScopes={0} is requesting permission to do the following:
/opt/jetty-9.4/temp/jetty-localhost-8081-oxauth.war-_oxauth-any-7004841259582193821.dir/webapp/WEB-INF/classes/oxauth_es.properties:authorize.requestingPermissionForScopes={0} esta solicitando permiso para lo siguiente:
/opt/jetty-9.4/temp/jetty-localhost-8081-oxauth.war-_oxauth-any-7004841259582193821.dir/webapp/WEB-INF/classes/oxauth.properties:authorize.requestingPermissionForScopes={0} is requesting permission to do the following:
/opt/jetty-9.4/temp/jetty-localhost-8081-oxauth.war-_oxauth-any-7004841259582193821.dir/webapp/WEB-INF/classes/oxauth_en.properties:authorize.requestingPermissionForScopes={0} is requesting permission to do the following:
```

If somehow oxAuth is forced to use this locale, authentication fails at the moment when consent dialogue should usually appear with next error message:
```
Caused by: java.lang.NumberFormatException: For input string: "client.getClientName()"
    at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65) ~[?:1.8.0_162]
```

I wasn't able to find some way to apply this locale in a conventional manner (but apparently one of our users did, only to encounter this issue; we may need to update docs on how to achieve this, as I didn't find this information there), instead I moved this dubious string fragment to the same line in the file `oxauth_en.properties` which is used by default - this produced exactly the same error stack trace as in this user's case.

**Comment by aliaksander-samuseu** [comment]
Next line may be related/need to be fixed:
https://github.com/GluuFederation/oxAuth/blob/master/Server/src/main/resources/oxauth_fr.properties#L16

**Comment by syntrydy** [comment]
Fixed in 3.1.4 branch
